examples/visualization/plot_topo_channel_epochs_image.py

```diff
@@ -39,6 +39,9 @@
 picks = mne.pick_types(raw.info, meg='grad', eeg=False, stim=True, eog=True,
                        exclude='bads')
 
+# Plot sensors in 3d
+raw.plot_sensors(kind='3d', ch_type='grad')
```

Comment:
Also, how do you think about the overlap between this and `plot_trans(info, trans=None, meg_sensors=True)`? It shoudl probably be mentioned in the `Notes` section. I'd say something about how `plot_trans` actually draws the sensors, but uses `mayavi` whereas this points out the locations and uses `matplotlib` or something. But it should be clear in the docstring of each function that the other function exists, and the functionality overlaps.